### ISSUE TITLE
Use behaviours to specific attachment support

### ISSUE THREAD

**Issue by mattss** [ploneintranet/ploneintranet.attachments#4]
This package seems like a good fit for a dexterity behaviour, rather than relying on other packages to manually provide the marker interface

**Comment by ebrehault** [comment]
+1

**Comment by reinhardt** [comment]
I agree and I'd like to look into it, but we'll have to see how it fits into the general plan.
